ECMA2Yaml/IntellisenseFileGen/IntellisenseFileGenHelper.cs

```diff
@@ -18,6 +18,7 @@ public class IntellisenseFileGenHelper
         static string _xmlDataFolder = @"G:\SourceCode\DevCode\dotnet-api-docs\xml";
         static string _docsetFolder = @"G:\SourceCode\DevCode\dotnet-api-docs";
         static string _outFolder = @"G:\ECMA2Yaml-output\GenerateIntellisense\_intellisense";
+        static string[] _monikers;
         static string _moniker = string.Empty;
```

Comment:
if there's only one moniker, we should also put it into the hashset, and use consistent logic below.